src/views/Board.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div>
+      <div
+      class="individual"
+      v-for="individual in individuals_with_ranking"
+      :key="individual.rank"
+      :individual="individual">
+        <p class="rank">{{ individual.rank }}</p>
+        <p class="name">{{ individual.username }}</p>
+        <p class="points">{{ individual.blocksCompleted }}</p>
+      </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Board',
+  props: {
+    individuals: {
+      type: Array,
+      required: true,
+    },
+  },
+  computed: {
+    individuals_with_ranking() {
+      const result = [];
```

Comment:
Generally, you want to avoid this pattern in JS of "Let this thing exist then we'll add to it" 

JS has pretty cool array methods, you can probably do a map on this.individuals to produce the same output, but more cleanly.